tools/dhis2-audit-data-extractor/COPYING

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Normally our software is released under the New BSD 3-Clause license (see [here](https://github.com/dhis2/dhis2-core/blob/master/LICENSE)), so maybe we could use that license instead?

Reply:
Yes we do, but in this repo there isn't an established license. Plus, I don't feel comfortable with the BSD clause for a software used as public good, so I'd prefer more open licenses like GPL3 or even [CC4r](https://constantvzw.org/wefts/cc4r.en.html).

So unless there is a strong need to align licenses, I'd keep GPL3.